src/matcha_ml/core/_validation.py

```diff
@@ -0,0 +1,97 @@
+""""Validation for core commands."""
+import os
+
+from matcha_ml.errors import MatchaInputError
+from matcha_ml.services import AzureClient
+
+# TODO: dynamically set both of these variables
+LONGEST_RESOURCE_NAME = "artifactstore"
+MAXIMUM_RESOURCE_NAME_LEN = 24
+MATCHA_STATE_PATH = os.path.join(".matcha", "infrastructure", "matcha.state")
```

Comment:
Could we use the state path in `constants.py` throughout matcha for a single source of truth? Though that may mean moving constants to core.

Reply:
You're right, I was meant to remove `MATCHA_STATE_PATH`, updated now